pages/item/_itemId/index.vue

```diff
@@ -410,8 +416,13 @@ export default Vue.extend({
   },
   methods: {
     ...mapActions('backdrop', ['setBackdrop', 'clearBackdrop']),
+    getItemizedSelect(objects: any[]) {
```

Comment:
```suggestion
    getItemizedSelect(objects: MediaSourceInfo[]) {

Reply:
I'm not sure about that, as this method 1) could be used with any type of list to make them compatible with the `v-select` (maybe this could be put in a mixin?) 2) if you check [L118](https://github.com/jellyfin/jellyfin-vue/pull/447/files#diff-bb43bf0fe8a0d41f72d41cd451108792debd7986e55b81c5064f466bd9389e67R118) you'll see takes a property of `item`

Reply:
Ah I only saw its uses for the media sources. Yeah a mixin is probably better, that way it can be used anywhere we use a `v-select`

Reply:
Okay I'll do it later today once I get a few minutes or tonight

Reply:
Done! :)